### PR TITLE
Fixed newline issue

### LINKED ISSUE #333: [333] Comment Newline
**Bug display and/or description**
image

When trying to add a newline in comment (Shift+Enter) it gets automatically sent. Maybe because defaultBehaviour is turned off?

**Reproduce**
- [ ] Go to a post
- [ ] Type something
- [ ] Shift+Enter to attempt creating a newline
- [ ] Comment gets sent already

**Branch**
333-hotfix-comment-newline

### PR DESCRIPTION
**Pull request by AjobK**
closes #333 

<br />

 **What is it supposed to do**  
When you press Shift+Enter it shouldn't send your comment. Only when you exclusively press enter.

<br />

 **How can we test your branch**  
- Type a comment
- Press shift+enter (Should make new line)
- Type more stuff
- Do all kinds of random key combinations if you want
- Press enter
- Post should be sent with newlines and all

<br />

 **Additional notes**  
Things to keep in mind regarding the PR that don't fit in the previous two headings. 

<br />

**1. General checklist**
- [x] a. Tested happy flow.
- [x] b. Tested unhappy flow.
- [x] c. No unexpected exceptions.
- [x] d. No code with high complexity. (Big O)
- [x] e. EOF newline.
- [x] f. No linter warnings or errors.

<br />

**2. If relevant, front-end checklist**
- [x] a. Responsive. (Desktop all the way to mobile 320px)
- [x] b. Using SCSS variables where relevant.
- [x] c. Relevant class names
- [x] d. No modified global SCSS properties, except if necessary and fully tested the impact.
- [x] e. Checked [caniuse](https://caniuse.com) for more modern CSS properties.

<br />

**3. If relevant, back-end checklist**
- [ ] a. No unhandled promise rejections.
- [ ] b. Seeders still functional.
- [ ] c. No primary-key violations.
- [ ] d. Normalization, at least [3NF](https://en.wikipedia.org/wiki/Database_normalization).
- [ ] e. Used guards (middleware) where necessary and tested if route is guarded.

<br />

**4. If relevant, test these browsers**
- [x] a. Chrome
- [x] b. Firefox
- [x] c. Opera
- [x] d. Edge
- [ ] e. Safari

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.

### CODE REVIEW

**Review by MrGooss — DISMISSED** [review]
LGTM!

File: frontend/src/components/commentEditor/index.js [hunk 1]
```diff
@@ -81,21 +81,17 @@ class CommentEditor extends Component {
   }
 
   keyBindingFn = (e) => {
```
Comment (danielordaan) [inline frontend/src/components/commentEditor/index.js]: While you're at it, I'd change variable 'e' to something more sensible, like pressedKeyEvent or something more to your liking.

**Review by danielordaan — CHANGES_REQUESTED** [review]
Besides the single comment, WFM, good fix

File: frontend/src/components/commentEditor/index.js (hunk 1 above)
Comment (AjobK) [reply frontend/src/components/commentEditor/index.js]: Agree

**Pull request merged**